### PR TITLE
Support different videos in hosted page template

### PR DESCRIPTION
**Pull request by kelvin-chappell**
This enables the [hosted template](https://github.com/guardian/frontend/blob/master/commercial/app/views/static/guardianHostedPage.scala.html) to show different videos.

/cc @Calanthe @lps88

### CODE REVIEW

File: commercial/app/views/hosted/episode1Video.scala.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+@()
+@import conf.Static
+
+<video
+data-duration="86"
+data-media-id="renault-car-of-the-future"
```
Comment (Calanthe) [inline commercial/app/views/hosted/episode1Video.scala.html]: The same media-id as in the teaser?

File: commercial/conf/routes [hunk 2]
```diff
@@ -3,49 +3,49 @@
...
 
 # Money merchandising components
-GET        /commercial/money/bestbuys.json                                                      controllers.commercial.MoneyOffers.renderBestBuys
-GET        /commercial/money/savings/:savingsType                                               controllers.commercial.MoneyOffers.savings(savingsType)
-GET        /commercial/money/current-accounts/:currentAccountType                               controllers.commercial.MoneyOffers.currentAccounts(currentAccountType)
-GET        /commercial/money/credit-cards/:creditCardType                                       controllers.commercial.MoneyOffers.creditCards(creditCardType)
-GET        /commercial/money/loans/:loanType                                                    controllers.commercial.MoneyOffers.loans(loanType)
+GET        /commercial/money/bestbuys.json                                      controllers.commercial.MoneyOffers.renderBestBuys
+GET        /commercial/money/savings/:savingsType                               controllers.commercial.MoneyOffers.savings(savingsType)
```
Comment (Calanthe) [inline commercial/conf/routes]: what does the colon mean?

Comment (jamesgorrie) [inline commercial/conf/routes]: It's a parameter you pass to the method:
https://www.playframework.com/documentation/2.5.x/JavaRouting#The-routes-file-syntax

**Comment by Calanthe** [comment]
👍

**Pull request merged**